src/index.js

```diff
@@ -10,20 +10,22 @@ const GoMulticastDNS = require('./compat')
 class MulticastDNS extends EventEmitter {
   constructor (options = {}) {
     super()
-    if (!options.peerInfo) {
-      throw new Error('needs a PeerInfo to work')
+
+    if (!options.peerId) {
+      throw new Error('needs own PeerId to work')
     }
 
     this.broadcast = options.broadcast !== false
     this.interval = options.interval || (1e3 * 10)
     this.serviceTag = options.serviceTag || 'ipfs.local'
     this.port = options.port || 5353
-    this.peerInfo = options.peerInfo
+    this.peerId = options.peerId
+    this.peerMultiaddrs = options.multiaddrs || []
     this._queryInterval = null
     this._onPeer = this._onPeer.bind(this)
```

Comment:
This function needs to be fixed. It still takes and emits `peerInfo`. Recommend doing a project case insensitive search for `peerinfo` to catch any other items.

Reply:
Probably just needs a rename to peerData to avoid confusion